app/controllers/project/volunteers_controller.rb

```diff
@@ -47,7 +47,7 @@ def delete
     logger.info "User has selected to delete volunteer ID: " \
                 "#{params[:volunteer_id]} from project ID: #{@project.id}"
 
-    volunteer = Volunteer.find_by_id(params[:volunteer_id])
+    volunteer = Volunteer.find_by(id: params[:volunteer_id])
```

Comment:
we could use `find` here which uses ID.
```suggestion
    volunteer = Volunteer.find(params[:volunteer_id])
```